partials/contacts/new.html

```diff
@@ -3,99 +3,104 @@ <h1>New Contact</h1>
 </div>
 
 <div class="col-md-offset-1"> 
+    <form name="newContactForm" novalidate>
 
+         
 
-    <fieldset class="container">
-        <legend class="col-md-2 control-label" >Name</legend> 
-            <fieldset class="form-group text-center">
-                <div class="col-md-4 inputGroupContainer">
-                    <div class="input-group">
-                        <span class="input-group-addon"><i class="fa fa-user-circle-o"></i></span>
-                        <input  name="first_name" placeholder="First Name" class="form-control"  type="text">
+        <fieldset class="container">
+            <legend class="col-md-2 control-label" >Name</legend> 
+                        <fieldset name="name" class="form-group text-center">
+                    <div class="col-md-4 inputGroupContainer">
+                        <div class="input-group">
+                            <span class="input-group-addon"><i class="fa fa-user-circle-o"></i></span>
+                            <input  name="first_name" placeholder="First Name" class="form-control"  type="text" required ng-model="contact.first_name" >
+                        </p>
+                        </div>
+                        <div class="input-group">
+                            <span class="input-group-addon"><i class="fa fa-user-circle-o"></i></span>
+                            <input name="last_name" placeholder="Last Name" class="form-control"  type="text" required ng-model="contact.last_name" >
+                            
+                        </div>
                     </div>
-                    <div class="input-group">
-                        <span class="input-group-addon"><i class="fa fa-user-circle-o"></i></span>
-                        <input name="last_name" placeholder="Last Name" class="form-control"  type="text">
-                    </div>
-                </div>
-            </fieldset>
-    </fieldset>
+                </fieldset>
+        </fieldset>
 
 
-    <fieldset class="container">
-        <legend class="col-md-2 control-label" >Contact</legend> 
-            <fieldset class="form-group text-center">
-                <div class="col-md-4 inputGroupContainer">
-                    <div class="input-group">
-                        <span class="input-group-addon"><i class="fa fa-envelope-o"></i></span>
-                        <input name="email" placeholder="E-Mail Address" class="form-control"  type="text">
-                    </div>
-                    <div class="input-group">
-                        <span class="input-group-addon"><i class="fa fa-mobile"></i></span>
-                        <input name="phone" placeholder="(845)555-1212" class="form-control" type="text">
+        <!-- <fieldset class="container">
```

Comment:
why is this code commented out?  comment it in or delete it

Reply:
Commented them out for testing purposes so that you wouldn't have to enter all of those fields just to test.